dash-pipeline/SAI/sai_api_gen.py

```diff
@@ -145,6 +147,18 @@ def table_with_counters(program, table_id):
             return 'true'
     return 'false'
 
+def fill_table_attrs(table_attrs, attr_names, action):
```

Comment:
The name is misleading. It's not all table attrs, only action parameters.

Reply:
Changed to fill_table_action_params